Training_Center_Task_11/Pages/InboxPage.cs

```diff
@@ -0,0 +1,59 @@
+﻿using OpenQA.Selenium;
+
+namespace Training_Center_Task_11.Pages
+{
+    public class InboxPage : MainPage
+    {
+        private IWebDriver _driver;
+
+        private IWebElement _composeButton;
+        private IWebElement _usernameLink;
+        private IWebElement _logoutLink;
+
+
+
+        const string COMPOSE_BUTTON_LOCATOR = "a[href='#compose']";
+        const string USERNAME_LINK_LOCATOR = "span[class='user-account__name']";
+        const string LOGOUT_LINK_LOCATOR = "a[aria-label='Log out']";
+
+
+
+        public InboxPage(IWebDriver driver) : base(driver)
+        {
+            this._driver = driver;
+            _composeButton = _driver.FindElement(By.CssSelector(COMPOSE_BUTTON_LOCATOR), Constants.WebDriverWaitTime);
```

Comment:
not a good way to find elements in constructor, cause driver will try to find elements during creation of object instance . It's better to find elemnt when you interact with it

Reply:
Moved elements out of constructors for all pages.